lib/full_text_search/batch_runner.rb

```diff
@@ -305,6 +315,11 @@ def project
         end
       end
 
+      def type
+        raw_type = super
+        Type[raw_type] if Type.available?(raw_type)
```

Comment:
存在しなかったら無視じゃなくてエラーでいいです。typoしたら全件対象になるのはしんどいっす。

```suggestion
        return nil if raw_type.blank?
        Type[raw_type]
```